specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -1856,6 +1856,97 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/namespaces/{namespaceName}/networkRuleSets/default": {
```

Comment:
seems exactly same set of changes..  so same feedback as in [https://github.com/Azure/azure-rest-api-specs/pull/5361](https://github.com/Azure/azure-rest-api-specs/pull/5361)